CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,2 @@
 Update Docker security practices (Add HEALTHCHECK, Use Anonymous User, Use two-stage build)
+Add: Support of multimeasure for MQTT and AMQP transport (#462)
```

Comment:
What about multimeasure functionality for MQTT and AMQP in IOTA-UL (https://github.com/telefonicaid/iotagent-ul/)?

My guess is that probably in that agent it's already implemented, but it would be great to have a confirmation in positive o negative sense.

Reply:
@fgalan 
I've verified the Multi measure functionality for MQTT and AMQP in IOTA-UL. It is working fine for both transports as well.
I've also added UT to check same functionality regarding AMQP. UT for MQTT is already written, i just executed and confirmed the successful execution.

Reply:
Ok. Great!

Thus, it is positively confirmed by your analysis and this comment thread is NTC.